lib/plugins/plugin/install.js

```diff
@@ -35,68 +37,61 @@ class PluginInstall {
       },
     };
     this.hooks = {
-      'plugin:install:install': () => BbPromise.bind(this).then(this.install),
+      'plugin:install:install': async () => this.install(),
     };
   }
 
-  install() {
+  async install() {
     const pluginInfo = pluginUtils.getPluginInfo(this.options.name);
     this.options.pluginName = pluginInfo[0];
     this.options.pluginVersion = pluginInfo[1] || 'latest';
 
-    return BbPromise.bind(this)
-      .then(this.validate)
-      .then(this.getPlugins)
-      .then((plugins) => {
-        const plugin = plugins.find((item) => item.name === this.options.pluginName);
-        if (!plugin) {
-          this.serverless.cli.log('Plugin not found in serverless registry, continuing to install');
-        }
-        return BbPromise.bind(this)
-          .then(this.pluginInstall)
-          .then(this.addPluginToServerlessFile)
-          .then(this.installPeerDependencies)
-          .then(() => {
-            const message = [
-              'Successfully installed',
-              ` "${this.options.pluginName}@${this.options.pluginVersion}"`,
-            ].join('');
-            this.serverless.cli.log(message);
-          });
-      });
+    await this.validate();
+    const plugins = await this.getPlugins();
+
+    const plugin = plugins.find((item) => item.name === this.options.pluginName);
+    if (!plugin) {
+      this.serverless.cli.log('Plugin not found in serverless registry, continuing to install');
+    }
+
+    await this.pluginInstall();
+    await this.addPluginToServerlessFile();
+    await this.installPeerDependencies();
+
+    const message = [
```

Comment:
I know it's not strictly part of `BbPromise` refactoring, but while we're at it - could you also simplify construction of that message? I don't think we need `join` here and putting it all in one line should be okay.